lib/views/schedule/schedule.dart

```diff
@@ -12,59 +14,48 @@ class Schedule extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Livestream Schedule'),
-        backgroundColor: ThcColors.darkBlue,
         actions: [
-          Visibility(
-            visible: userType.isAdmin,
-            child: IconButton(
-              icon: const Icon(Icons.add), // Choose your desired icon
-              onPressed: () {
-                // Add your onPressed callback function here
-                // This function will be called when the icon button is clicked
-                navigator.push(const ManageSchedule());
-              },
+          if (userType.isAdmin)
+            IconButton(
+              icon: const Icon(Icons.add),
+              onPressed: () => navigator.push(const ManageSchedule()),
             ),
-          ),
         ],
       ),
       body: SingleChildScrollView(
         child: Column(
           children: [
-            // First Header
-            Container(
-              margin: const EdgeInsets.only(top: 16.0),
-              padding: const EdgeInsets.all(16.0),
-              child: const Text(
-                'Active Livestreams',
+            const Padding(
+              padding: EdgeInsets.fromLTRB(16, 32, 16, 16),
+              child: Text(
+                'Active Livestream',
                 style: TextStyle(
                   fontSize: 24.0,
                   color: ThcColors.darkBlue,
                   fontWeight: FontWeight.bold,
                 ),
               ),
             ),
-            // First ListView inside a Card
-            Container(
-              margin: const EdgeInsets.all(16.0),
-              child: const Column(
-                children: [
-                  Card(
-                    color: ThcColors.green,
-                    margin: EdgeInsets.all(8.0), // Set margins for all sides
-                    child: ListTile(
-                      leading: FlutterLogo(size: 56.0),
-                      title: Text('Active Livestream'),
-                      subtitle: Text('April 3, 2024 12:00PM EST'),
-                    ),
-                  ),
-                  // Add more ListTiles as needed
-                ],
+            Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Card(
+                color: ThcColors.green,
+                margin: const EdgeInsets.all(8.0),
+                clipBehavior: Clip.antiAlias,
+                child: ListTile(
+                  onTap: () {
+                    context.read<NavBarIndex>().selectButton(NavBarButton.watchLive);
+                  },
```

Comment:
Oh yeah, almost forgot—I thought it'd be fun to go to the "watch live" page when you tap on the current livestream.